Tests/presentations.lisp

```diff
@@ -105,4 +105,8 @@
   (expect-t-t '(or integer symbol) '(or symbol integer))
 
   (expect-t-t '(or real complex) 'number)
-  #+nil (expect-t-t 'number '(or real complex)))
+  (expect-t-t 'number '(or real complex)))
+
+(test presentations.type-relations.5
+  (let ((type (expand-presentation-type-abbreviation '(member "a" 3))))
+    (expect-t-t type '(or string integer))))
```

Comment:
Does fiveam support something like "expected failures"? Otherwise having more failing tests makes test suite less useful.

Reply:
it doesn't, but it would be nice to have in the fiveam. As of the usefulness, I suppose it depends on what do you expect from tests: to show defects or to show regressions.

Reply:
I use automatic execution the test suite and result analysis for spotting regressions but I agree that indicating defects is valuable too. So yak shaving in fiveam is the solution, then?